source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -868,56 +874,13 @@ export const handleHardwareWalletRequests = async (
       throw e;
     }
   });
-  // @ts-ignore ts-migrate(2345) FIXME: Argument of type '(params: TrezorSignTransactionRe... Remove this comment to see the full error message
-  signTransactionTrezorChannel.onRequest(async (params) => {
-    const {
```

Comment:
This whole block of code was redundant, we were just reassigning parameters instead of providing them in the final shape on the renderer process, and passing the object as-is on the main process.